zproject/backends.py

```diff
@@ -1470,6 +1483,133 @@ class GitLabAuthBackend(SocialAuthMixin, GitLabOAuth2):
     # we just use the primary email address, which is always verified.
     # (No code is required to do so, as that's the default behavior).
 
+@external_auth_method
+class AppleAuthBackend(SocialAuthMixin, AppleIdAuth):
```

Comment:
Can you add a block comment here explaining the basic flow of `AppleIdAuth`?  I'm not particularly familiar with JWT, and in any case we need this to be readable be someone working on Zulip's authentication code who isn't even if I am.